src/controllers/challenge.controllers.js

```diff
@@ -15,4 +15,71 @@ export const createChallenge = async (req, res) => {
   }
 };
 
-export default { createChallenge };
+// Get all challenges
+export const getChallenges = async (req, res) => {
+  try {
+    const data = await Challenge.find();
+    // TODO: Filter out challenges that the creater has left
```

Comment:
Ticket created for this TODO: https://app.clickup.com/t/86b0vg3m8

Reply:
Thanks!